pkgs/development/tools/metals/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, lib, fetchurl, coursier, jdk, jre, makeWrapper }:
+
+let
+  baseName = "metals";
+  version = "0.3.1";
```

Comment:
v0.3.2 is out. Once this PR is merged we can update our release process (https://github.com/scalameta/metals/blob/master/RELEASING.md) to make sure this nix installation stays up-to-date.

Reply:
I updated to 0.3.2 a few days ago. I know that there was a 0.3.3 patch, but I'm not sure whether I should update this PR or if it's easier to get it merged if we freeze it and submit updates in future PRs.